src/main/java/no/nav/fo/veilarbregistrering/oppgave/OppgaveMetrikker.java

```diff
@@ -0,0 +1,15 @@
+package no.nav.fo.veilarbregistrering.oppgave;
+
+import no.nav.fo.veilarbregistrering.bruker.GeografiskTilknytning;
+import no.nav.metrics.Event;
+import no.nav.metrics.MetricsFactory;
+
+class OppgaveMetrikker {
+
+    static void rapporter(GeografiskTilknytning geografiskTilknytning, String tildeltEnhetsnr) {
+        Event event = MetricsFactory.createEvent("arbeid.registrering.oppgave");
```

Comment:
Vi må ta en runde på hvordan vi utformer id for metrikkene våre. Jeg mener at alle metrikker bør starte med applikasjonsnavnet

Reply:
Jeg ville gjerne ha prefikset med arbeid :)

Reply:
Hva er hensikten med det?

Reply:
Jeg ser for meg at det er mulig å fange opp events innenfor område, og når oppnavnet er lite forklarende, kan det være fint å si litt om hvor det hører hjemme.

Reply:
Mitt poeng handler egentlig kun om at appnavnet bør være inkludert i metrikkene så det er enkelt for oss som utvikler produktet å vite hvor de kommer i fra.

Om metrikken også inneholder 'arbeid' er egentlig det samme for meg, men jeg er generelt skeptisk til unødvendig lange ID'er ettersom dette gjør spørringene på disse metrikkene tunge å lese. Det er derfor jeg spør 😊

Hva er hensikten med å fange opp alle events under `arbeid`? Hva skal det brukes til?